packages/core/help-text.scss

```diff
@@ -0,0 +1,17 @@
+@import "./variables/all";
+@import "./mixins/all";
+
+.jkl {
+    &__text {
```

Comment:
Better nameing suggestion?

Reply:
Alternatively, we could do something a bit more abstract that would let us use component-specific BEM naming in the different form fields:
```scss
[class$="__message"] {
    // common styles
    &--help {
        color: $svart;
    }
    &--error {
        color: $error-color;
    }
}
```
Then in `text-field` we could do
```jsx
<span className="jkl-text-field__message jkl-text-field__message--error">Feilmelding</span>
```
and similarly in  `dropdown`
```jsx
<span className="jkl-dropdown__message jkl-dropdown__message--error">Feilmelding</span>
```
and so on. The discoverability for the developers might be a bit worse, though..

Reply:
I do like the last one, but it seems a bit magical i think.

How about `jkl-form-support-text`?